node_modules/col-canvas/lib/poller.js

```diff
@@ -464,9 +469,11 @@ var handleAssignmentCategory = function(ctx, users, categories, assignment, call
       return callback();
     });
 
-  // Create a category for an assignment if it doesn't exist yet
+  // Create a category for an assignment if it doesn't exist yet.
+  // By default, the category will be marked as not visible
   } else {
-    CategoriesAPI.createCategory(ctx, assignment.name, assignment.id, assignment.name, function(err, newCategory) {
+    var enableAssignmentCategories = opts.enableAssignmentCategories  || false;
```

Comment:
@nicolaasmatthijs for what it's worth, the line above seems unnecessary. Your logic in `CategoriesAPI.createCategory` would convert falsey to false.